### DISCUSSION TITLE
More granular Core Clock Limit?

### DISCUSSION THREAD

**Discussion by XzRaiderzX** [seerge/g-helper#1923] [Q&A] [answered]
Is there anyway to make the steps for Core Clock Limit the lowest it can be which is steps of 15, through the config maybe? It would make afterburner completely obsolete for dGPU undervolting.

Also as a sidenote, I presume there's no way to turn on "Dynamic refresh rate" through GHelper right?

**Comment by seerge** [comment]
@XzRaiderzX i have intentionally set it to round values, to keep things more simple. Is there any practical reason to set it so specifically ?

For refresh rate - G-Helper just uses windows API, and there I don't see such option

**Reply by XzRaiderzX** [reply]
![Screenshot 2024-01-21 010000](https://github.com/seerge/g-helper/assets/156614762/53447925-45a5-4e6e-9d0e-c61b222ed208)

Strange that you don't see it under Windows API, because I only found this setting at all in advanced display settings.

Regarding the clore clock limit, I thought about it and I guess there isn't really a practical benefit yeah lol :)
It just would be a nice power user setting since GHelper already has the ability to switch GPU profiles for AC and Battery, something afterburner was never capable of doing.

**Reply by seerge** [reply, marked as answer]
@XzRaiderzX windows API in that matter is quite ancient, so it doesn't have new fancy things awailable via windows own settings page :)

For clock, i think current steps are good enough.

And you already can set different GPU profiles per AC / Battery. As GPU settings are part of mode settings, and app remembers last used mode on AC / Battery and switches it automatically
